examples/screen_cast_pw.rs

```diff
@@ -0,0 +1,218 @@
+use std::os::fd::{IntoRawFd, OwnedFd};
+
+use ashpd::desktop::{
+    screencast::{CursorMode, Screencast, SourceType, Stream as AshStream},
+    PersistMode,
+};
+use pipewire as pw;
+use pw::{properties::properties, spa};
+
+use spa::pod::Pod;
+
+struct UserData {
+    format: spa::param::video::VideoInfoRaw,
+}
+
+async fn open_portal() -> ashpd::Result<(AshStream, OwnedFd)> {
+    let proxy = Screencast::new().await?;
+    let session = proxy.create_session().await?;
+    proxy
+        .select_sources(
+            &session,
+            CursorMode::Hidden,
+            SourceType::Monitor.into(),
+            false,
+            None,
+            PersistMode::DoNot,
+        )
+        .await?;
+
+    let response = proxy.start(&session, None).await?.response()?;
+    let stream = response.streams().first().unwrap().to_owned();
```

Comment:
Should maybe check first if there are any streams? but as it is an example, maybe that is fine

Reply:
I guess I can turn the unwrap into an expect with an error message

Reply:
that would be better indeed